docs.html

```diff
@@ -32,11 +32,17 @@
             <li>
                 <a class="nav-link md-ht-1" id="dark-mode-toggle" role="button" aria-label="toggle dark mode"><i class="fas fa-moon"></i></a>
             </li>
+            <button class="btn-hamburger btn btn-primary" data-js="toggle-sidebar"><i class="fas fa-bars"></i></button>
+              <!-- <div class="btn-hamburger md-ht-1" data-js="toggle-sidebar">
+                <i class="fas fa-bars"></i>
+              </div> -->
```

Comment:
Can remove this commented code from here.

Reply:
Hey, @Maianki I have removed the comments.

Reply:
Thanks